openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -91,11 +93,12 @@ def create_jwt_token_dict(token_dict, oauth_adapter, use_asymmetric_key=None):
     )
 
     jwt_token_dict = token_dict.copy()
-    # Note: only "scope" is not overwritten at this point.
+    # Note: only "refresh_token" is not overwritten at this point.
```

Comment:
Here's a possible code comment trail if we decide to ignore #2 from this comment: https://github.com/openedx/edx-platform/pull/33455/files#r1372861173
```suggestion
    # Note: only "refresh_token" is not overwritten at this point.
    # At this time, the user_id scope added for grant type password is only added to the
    # JWT, and is not added for the DOT access token or refresh token, so we must override
    # here. If this inconsistency becomes an issue, then the user_id scope should be
    # added earlier with the DOT tokens, and we would no longer need to override "scope".
```